lib/spack/spack/bootstrap/environment.py

```diff
@@ -44,6 +44,7 @@ def spack_dev_requirements(cls) -> List[str]:
             black_root_spec(),
             flake8_root_spec(),
             pytest_root_spec(),
+            pyruff_root_spec(),
```

Comment:
Why pyruff instead of ruff? All the other formatters don't have a py prefix.

Reply:
Because I am bad at reading mostly :p I'll update that once I figure out why the bootstrap test is angry.